ansible/molecule-playbook.yml

```diff
@@ -2,6 +2,8 @@
 
 # Check the networking is setup as expected
 - hosts: all
+  roles:
+  - role: openmicroscopy.selinux-utils
```

Comment:
You shouldn't need this as it'll be installed in the standard playbook that you're testing.